README.md

```diff
@@ -1,2 +1,7 @@
 # v4.loopback.io
-LoopBack 4 Web Site
+![LoopBack 4 logo](http://loopback.io/images/branding/mark/blue/loopback.jpg width=100)
```

Comment:
When I click `View` when reviewing, there's no image showing up. Will it work once it's merged?

Reply:
i've just fixed it :)